plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -71,10 +70,30 @@ struct _FuPxiDevice {
 	guint16		 mtu_size;
 	guint16		 prn_threshold;
 	guint8		 spec_check_result;
+	struct hidraw_devinfo	hid_raw_info;
```

Comment:
does this need to be in the device privdata? It only seems read and used in one place...